public/datasets/state_demographics.json

```diff
@@ -0,0 +1,227 @@
+    },
+    {
+      "type": "continuous",
+      "id": "Income.Persons Below Poverty Level"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Building Permits"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Foreign Born"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Land Area"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Language Other than English at Home"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Living in Same House +1 Years"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Manufacturers Shipments"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Mean Travel Time to Work"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Percent Female"
+    },
+    {
+      "type": "continuous",
+      "id": "Miscellaneous.Veterans"
+    },
+    {
+      "type": "continuous",
+      "id": "Population.2010 Population"
+    },
+    {
+      "type": "continuous",
+      "id": "Population.2014 Population"
+    },
+    {
+      "type": "continuous",
+      "id": "Population.Population Percent Change"
+    },
+    {
+      "type": "continuous",
+      "id": "Population.Population per Square Mile"
+    },
+    {
+      "type": "continuous",
+      "id": "Sales.Accommodation and Food Services Sales"
+    },
+    {
+      "type": "continuous",
+      "id": "Sales.Merchant Wholesaler Sales"
+    },
+    {
+      "type": "continuous",
+      "id": "Sales.Retail Sales"
+    },
+    {
+      "type": "continuous",
+      "id": "Sales.Retail Sales per Capita"
+    },
+    {
+      "type": "continuous",
+      "id": "Employment.Firms.American Indian-Owned"
+    },
+    {
+      "type": "continuous",
+      "id": "Employment.Firms.Asian-Owned"
+    },
+    {
+      "type": "continuous",
+      "id": "Employment.Firms.Black-Owned"
+    },
+    {
+      "type": "continuous",
+      "id": "Employment.Firms.Hispanic-Owned"
+    },
+    {
+      "type": "continuous",
+      "id": "Employment.Firms.Native Hawaiian and Other Pacific Islander-Owned"
+    },
+    {
+      "type": "continuous",
+      "id": "Employment.Firms.Total"
+    },
+    {
+      "type": "continuous",
+      "id": "Employment.Firms.Women-Owned"
+    }
+  ],
+  "defaultLabelColumn": "Housing.Homeownership Rate",
+  "card": {
+    "description": "Summarized demographic data about individual US states (and the District of Columbia) from 2015.",
+    "source": "Corgis Dataset Project (https://corgis-edu.github.io/corgis/csv/state_demographics/)",
+    "last_updated": "2016",
+    "context": {
+      "created_by": "US Census",
+      "preparation": true,
+      "confidentiality": false,
+      "subgroupIdentifiers": true,
+      "potentialUses": "Predict a state's home ownership rate based on median household income and the number of persons below the poverty line.",
+      "potentialMisuses": ""
```

Comment:
Is this intentionally blank? Seems like there actually are some potential misuses for this dataset.

Reply:
Brent made this metadata file. I agree that we should add something here - I'll see if he has some thoughts and include it in a later update